cart-service/src/tests/service-endpoint.tests.ts

```diff
@@ -244,6 +244,125 @@ test('should return action to set coupon codes on cart when Dovetech service ret
   });
 });
 
+test('should return set commitId action when type is Order', async () => {
+  const currencyCode = 'EUR';
+  const amountOff = 10;
+  const total = 22.97;
+
+  const ctCart = orderWithEvaluationResult as CartOrOrder;
+
+  const amountOffBasketAction: AmountOffAction = buildAmountOffAction(
+    DoveTechActionType.AmountOffBasket,
+    amountOff
+  );
+
+  const dtResponse = new DoveTechResponseBuilder()
+    .addCommitId('123')
+    .addAction(amountOffBasketAction)
+    .addLineItem({
+      totalAmountOff: amountOffBasketAction.amountOff,
+      total: total,
+      actions: [
+        {
+          id: amountOffBasketAction.id,
+          subItemId: 0,
+          amountOff: amountOffBasketAction.amountOff,
+        },
+      ],
+    })
+    .build();
+
+  fetchMock.mockResponseOnce(JSON.stringify(dtResponse));
+
+  const response = await postCart(ctCart);
+
+  expect(response.status).toBe(200);
+
+  expect(response.body).toEqual({
+    actions: [
+      buildSetCustomTypeAction(  dtResponse, currencyCode, '[]' ),
+    ],
+  });
+});
+
+test('should return no discount actions when type is Order', async () => {
```

Comment:
This test is the same as the "should return set commitId action when type is Order" test, so I don't think it's needed?

Reply:
The idea was to test that the direct discount action was actually never passed...  
![{09711C23-53A3-409E-BED8-52069CBEBEC6}](https://github.com/user-attachments/assets/78609579-d501-4437-b479-fc6fc185b3ca)

but guess that does not make sense

Reply:
Yeah, that's actually covered by the "should return set commitId action when type is Order" test because the Dovetech service returns an action, but it isn't returned as a Direct Discount.